cmd/kuberhealthy/main.go

```diff
@@ -105,8 +105,9 @@ func init() {
 	// set env variables into config if specified
 	externalCheckURL, err := getEnvVar(KHExternalReportingURL)
 	if err != nil {
-		cfg.ExternalCheckReportingURL = externalCheckURL
+		log.Infoln("KH_EXTERNAL_REPORTING_URL environment variable not set: ", err, "Attempting to use default value")
 	}
+	cfg.ExternalCheckReportingURL = externalCheckURL
```

Comment:
it looks like we're doing this validation twice. Once here, and then another time in lines 130-136. I'd rather try and combine them if possible..

Reply:
If the env var is not set `externalCheckURL` would be empty... and then go through the second validation below, which sort of seems repetitive?